File: bank-service/src/main/java/rs/edu/raf/repository/accounts/CurrentBankAccountRepository.java
package rs.edu.raf.repository.accounts;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;
import rs.edu.raf.model.accounts.CurrentBankAccount;

import java.util.Optional;

@Repository
public interface CurrentBankAccountRepository extends JpaRepository<CurrentBankAccount,Long> {
    Optional<CurrentBankAccount> findByOwner(Long owner);
}

File: bank-service-kotlin/src/main/kotlin/org/example/bankservicekotlin/repository/accounts/CurrentBankAccountRepository.kt
package org.example.bankservicekotlin.repository.accounts

import org.example.bankservicekotlin.model.accounts.CurrentBankAccount
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.util.*

@Repository
interface CurrentBankAccountRepository: JpaRepository<CurrentBankAccount, Long> {
    fun findByOwner(owner: Long): Optional<CurrentBankAccount>
}